ovos_PHAL_plugin_system/__init__.py

```diff
@@ -210,16 +210,23 @@ def handle_ssh_disable_request(self, message):
             self.gui.show_page(page)
 
     def handle_ntp_sync_request(self, message):
-        ntp_sync()
-        # NOTE: this one defaults to False
-        # it is usually part of other groups of actions that may
-        # provide their own UI
-        if message.data.get("display", False):
-            page = join(dirname(__file__), "ui", "Status.qml")
-            self.gui["status"] = "Enabled"
-            self.gui["label"] = "Clock updated"
-            self.gui.show_page(page)
-        self.bus.emit(message.reply('system.ntp.sync.complete'))
+        # Check to see what service is installed
+        if check_service_installed('ntp'):
+            ntp_sync()
+        elif check_service_installed('systemd-timesyncd')
+            timesync_sync()
```

Comment:
go we move both these methods from ovos utils to this plugin and deprecate them in ovos-utils?